aim/aim_manager.py

```diff
@@ -98,7 +100,6 @@ class AimManager(object):
                      api_service_graph.DeviceClusterInterfaceContext,
                      api_service_graph.ServiceRedirectMonitoringPolicy,
                      api_service_graph.ServiceRedirectHealthGroup,
-                     api_infra.OpflexDevice,
```

Comment:
oh it was just being moved to line 82 so that all the classes under api_infra are grouped together. Whoever added it didn't follow the convention so.....